shared/src/androidMain/kotlin/co/nimblehq/kmm/template/di/module/PlatformModule.kt

```diff
@@ -0,0 +1,8 @@
+package co.nimblehq.kmm.template.di.module
+
+import org.koin.core.module.Module
+import org.koin.dsl.module
+
+actual fun platformModule(): Module = module {
+
```

Comment:
Please add TODO here

Reply:
Updated in https://github.com/nimblehq/kmm-templates/pull/25/commits/fdd5696df9c927e082a25dfc5261a991eb8d88a6